test/types/string/parIter/cyclicUTF8.chpl

```diff
@@ -0,0 +1,14 @@
+use CyclicDist;
+
+config const n = 10;
+
+var s= "çş"*n;
+
+var arr = newCyclicArr(0..<2*n, string);
+
+forall (elem, char) in zip(arr, s) {
+  elem = char;
+}
+
+writeln(arr);
```

Comment:
Since this is the only test that checks a non-dense follower, and there are effectively three iterators (utf8, ascii, bytes), it might make sense to have it also follow an ascii string and a bytes to lock in the behavior?